docs/guides/qiskit-addons-mpf.ipynb

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
<p>That final pip install command does not work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>